### DISCUSSION TITLE
how to add a email notification to even and cron triggers ?

### DISCUSSION THREAD

**Discussion by abythachet** [hasura/graphql-engine#6812] [Question]
I have some triggers in my Hasura I need to add email notification if it fails,

**Comment by jtkelley** [comment]
I was looking at this also.   All I could come up with is using the hdb_catalog tables in your database. The hdb_cron_event_invocation_logs table gives the status code. So you could set up your monitoring to watch that. You could even use DB triggers to do some handling.   It would be nice to have built-in webhooks or notifications.      I would also like the ability to disable/enable cron triggers.  Meh! ...  Good chance I end up using a different solution for scheduled tasks.

**Reply by jtkelley** [reply]
There are similar log tables for the other events.
